emailer.py

```diff
@@ -70,6 +76,73 @@ class Emailer(object):
         'from_address',
     ]
 
+    ##### Public Interface / Magic Methods ###################################  # noqa: E266
+    def __init__(self, smtp_username, smtp_password, smtp_host, smtp_port, from_name, from_address, templates):
+        config_errors = []
+        required_parameters = ('smtp_username', 'smtp_password', 'smtp_host', 'smtp_port', 'from_name', 'from_address')
+        for param_name in required_parameters:
+            if not locals()[param_name]:
+                config_errors.append(param_name)
+
+        if config_errors:
+            msg = "Emailer instantiated with missing params: " + ", ".join(config_errors)
+            raise CannotLoadConfiguration(msg)
```

Comment:
Switched out the individual `if` statements for this, to raise a more comprehensive error message.